tests/acceptance/features/apiSharingNg/listPermissions.feature

```diff
@@ -196,3 +196,27 @@ Feature: List a sharing permissions
         }
       }
       """
+
+
+  Scenario: user send share invitation for all allowed roles defined in permission lists for a file
+    Given user "Alice" has uploaded file with content "hello text" to "textfile.txt"
+    And user "Brian" has been created with default attributes and without skeleton files
+    When user "Alice" gets permissions list for file "textfile.txt" of the space "Personal" using the Graph API
+    Then the HTTP status code should be "200"
+    And user "Alice" should be able to send share invitation with all allowed permission roles from the above response:
```

Comment:
I see that this test is failed. Hmm we're getting all permissions for `textfile.txt`, but not all we can use for share invitation. I guess that some roles we use for creating link

that is cool that we can see in log what is wrong: 
<img width="855" alt="image" src="https://github.com/owncloud/ocis/assets/84779829/fe09a153-5b8d-4ca2-8afe-0acf27b67692">

`with all allowed permission` - looks like something abstract. which role used test? 
Is there any way to log all the checks, like this, when error happens: 

```
Share invitation for resource 'textfile.txt' with role 'Uploader' failed and was not allowed.
Share invitation for resource 'textfile.txt' with role 'Viewer' was allowed.
Share invitation for resource 'textfile.txt' with role 'Editor' failed and was not allowed.
Share invitation for resource 'textfile.txt' with role 'Editor' failed and was not allowed.
```

Reply:
Hi @ScharfViktor This test check whether all the allowed roles in the list permissions works fine for sending invitations or not. From the any of the allowed roles, if we are unable to send share invitations then the tests fails with the above information you mentioned.